docs/Edit_Page.md

```diff
@@ -0,0 +1,40 @@
+# Edit page
+## Link: 
+mwet.lbl.gov/pages/\<specific_uid_here\>
+
+## Purpose
+This page displays a specific document and its corresponding references for the purpose of viewing and editing. 
+
+## Title
+The title of the document will be at the top of the page. You can edit the title by double clicking on it, or clicking the pencil icon. 
+
+## Metadata
+In the upper right corner you can see a card that will display the date created and the creator. If the document has been edited then it will also display the date it was last edited at and the name of the person who edited it.
+
+## View past versions
+This button will take you to the ["Versioned view"](#) of this page.
+
+## The Document
+There will be a box that displays the actual content of the document. To edit the document, double click anywhere inside the box, or click on the pen icon in the upper left corner of the document. Hovering over any parenthetical references in the document should display a small pop up next to the cursor that will contain the APA citation of the corresponding reference. Clicking on any in-text citations will bring you down to 
+    
+ ## Editing the document
+ * ### Editor toolbar:
+    The use of each button will be displayed by a tooltip if the the user hovers their mouse over the icon.
+ * ### Insert Reference:
+    This button will open up the "Insert Reference" pop up. Which is explained later in this document.
+* ### Save/Cancel:
+    The "Save" button will be disabled unless some change has been made to the document. The "Cancel" button will discard all changes made to the document since the last edit. 
+
+## Inserting in-text citations
```

Comment:
This section should be merged with the similar section in Tutorials

Reply:
Do you mean the Inserting in text citations section or the save/cancel section?